.sdkmanrc

```diff
@@ -2,3 +2,4 @@
 # Add key=value pairs of SDKs to use below
 java=17.0.17.fx-zulu
 ant=1.10.14
+gradle=9.2.1
```

Comment:
I don't think this change is necessary. The gradle version is handled by the gradle wrapper.